src/views/v2/rsvp/ari/form.php

```diff
@@ -17,7 +17,6 @@
  *
  * @version 5.0.0
  */
-
```

Comment:
Is the removal of this line necessary?

Reply:
Yes, because we are not modifying anything in that template anymore.

Reply:
Sorry @redscar I see what you mean with the snapshots failing. I have added the new line spacing to see if this fixes it.